.cirrus.yml

```diff
@@ -40,15 +40,15 @@ task:
   name: tests-macos-0
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_PR == ''
   script: scripts/verify_tests_on_master.sh --shards 2 --shard-index 0
-  osx_instance:
-    image: mojave-base
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-xcode:14
```

Comment:
Or: does `macos-ventura-vanilla:13.1` work? Not sure if this repo needs `brew` installed.
https://github.com/cirruslabs/macos-image-templates/pkgs/container/macos-ventura-vanilla

Reply:
`macos-ventura-vanilla:13.1` failed with a different error:

`xcode-select: note: no developer tools were found at '/Applications/Xcode.app', requesting install. Choose an option in the dialog to download the command line developer tools.`

(https://cirrus-ci.com/task/5306558125441024?logs=main#L17)